CMakeLists.txt

```diff
@@ -69,6 +69,13 @@ IF(MSVC)
     add_definitions(-D_CRT_SECURE_NO_WARNINGS)
 ENDIF(MSVC)
 
+IF(WIN32)
+    # Explicitly state minimum Windows version to compile against
+    # This affects which APIs that are available in for example winsock
+    # 0x0601 == Windows 7
+    add_definitions(-D_WIN32_WINNT=0x0601)
```

Comment:
Windows 7 is end of life. Same with 8, 8.1, and server 2012.

This should be either Windows 10, or windows server 2016 to properly reflect supported operating systems.